### DISCUSSION TITLE
[DE Number] Support for German numbers in JS/Python

### DISCUSSION THREAD

**Discussion by PeterSchoell** [microsoft/Recognizers-Text#1516] [General]
Attempt to convert the Recognizer result to an integer. This SHOULD works for "a dozen", "twelve", "12", and so on. Correspondingly for the German terms like "Zwölf" or "Zehn"

Javascript:
const results = Recognizers.recognizeNumber(input, Recognizers.Culture.German);

This works fine for English culture, but not for German culture.

Proposed change
Add German prebuilt entities

See also: https://github.com/Microsoft/BotBuilder/issues/3540

**Comment by tellarin** [comment]
Support for German currently only exists in .NET. Any help in propagating to JavaScript is very welcome.
